pages/gallery.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title> Gallery </title>
+  <link href="/stylesheets/gallery.css" rel="stylesheet" />
+  <link href="/stylesheets/style.css" rel="stylesheet" />
 </head>
+
+
 <body>
-  
+  <header>
+    <h1>Gallery</h1>
+    <a class="header-links" href="index.html">Home</a> <a class="header-links" href="facts.html">Facts</a>
+  </header>
+  <div class="container">
+    <img src="/images/little_kids_reading.jpg" alt="Kids reading" />
+    An older child reading to a toddler on a couch!
```

Comment:
This is some floating text, I think you may be better served throwing these in a `<p>` or `<span>` tag